openai-client/client/src/commonMain/kotlin/com/xebia/functional/openai/infrastructure/ApiClient.kt

```diff
@@ -35,6 +35,7 @@ open class ApiClient(val baseUrl: String) {
       {
         it.install(ContentNegotiation) { json(jsonBlock) }
         it.install(HttpTimeout)
+        it.install(Logging) { level = LogLevel.NONE }
```

Comment:
Should that be configurable? 🤔 Not blocked for this PR.

Reply:
Good catch! Yes, this should be configurable. We are going to address this in another PR because this PR is blocking other stuff